CAM/PlaySynchronizedCamAnim.md

```diff
@@ -5,7 +5,7 @@ ns: CAM
 
 ```c
 // 0xE32EFE9AB4A9AA0C 0x9458459E
-BOOL PLAY_SYNCHRONIZED_CAM_ANIM(Any p0, Any p1, char* animName, char* animDictionary);
+BOOL PLAY_SYNCHRONIZED_CAM_ANIM(Cam camera,int scene, char* animName, char* animDictionary);
```

Comment:
Add space between arguments and would be nice.
i.e. `Cam camera, int scene`
-----------------^